jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -1311,10 +1315,18 @@ protected HttpClient getHttpClient(Integer readTimeout,
 
             HttpClientBuilder clientBuilder = HttpClientBuilder.create()
                     .setDefaultRequestConfig(config);
+            
+            
+            if (Boolean.getBoolean(USE_SYSTEM_PROPERTIES)) {
```

Comment:
done only when HTTP_CLIENT_API_43 client.... when this is false does not use the proxy.

Reply:
Changed